dev/tasks/tasks.yml

```diff
@@ -1392,6 +1392,7 @@ tasks:
         PYTHON: "{{ python_version }}"
         PANDAS: "{{ pandas_version }}"
         NUMPY: "{{ numpy_version }}"
+        PARQUET_REQUIRE_ENCRYPTION: "OFF"
```

Comment:
Can we do this just for one of these builds, not all of them?

Reply:
It is. I suggest to remove `cache_leaf` jinja variable and just set `flags: --no-leaf-cache` for all cases.